test/support/dev/billing/dev_paddle_api_mock.ex

```diff
@@ -9,7 +9,14 @@ defmodule Plausible.Billing.DevPaddleApiMock do
   @prices_file_path Application.app_dir(:plausible, ["priv", "plan_prices.json"])
   @prices File.read!(@prices_file_path) |> Jason.decode!()
 
-  def prices(), do: @prices
+  def all_prices() do
+    enterprise_plan_prices =
+      Plausible.Billing.EnterprisePlan
+      |> Plausible.Repo.all()
+      |> Map.new(fn ep -> {ep.paddle_plan_id, 123} end)
```

Comment:
```suggestion
    Repo.all(from p in Plausible.Billing.EnterprisePlan, select: {p.paddle_plan_id, 123})
```

w/ formatter applied